cms/templates/js/content-group-details.underscore

```diff
@@ -41,17 +41,18 @@
 <% if (showContentGroupUsages) { %>
   <div class="collection-references wrapper-group-configuration-usages">
     <% if (!_.isEmpty(usage)) { %>
-      <h4 class="intro group-configuration-usage-text"><%= gettext('This content group is used in:') %></h4>
+      <h4 class="intro group-configuration-usage-text"><%- gettext('This content group is used in:') %></h4>
       <ol class="usage group-configuration-usage">
         <% _.each(usage, function(unit) { %>
           <li class="usage-unit group-configuration-usage-unit">
-            <p><a href=<%= unit.url %> ><%= unit.label %></a></p>
+            <p><a href=<%= unit.url %> ><%- unit.label %></a></p>
```

Comment:
@robrap I'll think we'll be able to escape this without breaking the URL, it doesn't look to have any query params.

<img width="1428" alt="screen shot 2016-03-14 at 1 46 38 pm" src="https://cloud.githubusercontent.com/assets/7373924/13753892/54fd2c1a-e9eb-11e5-930a-8ca679035f7d.png">
Clicking the link works as expected, the link as it appears in HTML is `/container/block-v1:edX+DemoX+Demo_Course+type@vertical+block@vertical_0270f6de40fc`.

I believe the link is being populated in this code https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/course_group_config.py#L112-L117